### ISSUE TITLE
 SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder"

### ISSUE THREAD

**Issue by tonyyang924** [MicroUtils/kotlin-logging#47]
i added below code to build.gradle
`
implementation 'io.github.microutils:kotlin-logging:1.5.9'
`

then I compiler & run show this message:
`
09-02 00:31:12.297 28733-28733/com.example.tonyyang.tonyptt W/System.err: SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
    SLF4J: Defaulting to no-operation (NOP) logger implementation
    SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
`

**Comment by oshai** [comment]
This means slf4j could not bind to actual logging. from the link of details: `Placing one (and only one) of slf4j-nop.jar slf4j-simple.jar, slf4j-log4j12.jar, slf4j-jdk14.jar or logback-classic.jar on the class path should solve the problem.`  
See this for more details: http://saltnlight5.blogspot.com/2013/08/how-to-configure-slf4j-with-different.html  
Please reopen if you need more assistance.

**Comment by temirov** [comment]
If this project is not self sufficient then the installation steps should describe a minimal working example. As it stand now it doesn't look like adding `implementation 'io.github.microutils:kotlin-logging:1.5.9'` will allow logging.

**Comment by oshai** [comment]
the readme sate that:
```
Download
Important note: kotlin-logging depends on slf4j-api (in the JVM artifact). In runtime, it is also required to depend on a logging implementation. More details in [how-to-configure-slf4j](http://saltnlight5.blogspot.co.il/2013/08/how-to-configure-slf4j-with-different.html). And an excellent detailed explanation in [a-guide-to-logging-in-java](https://www.marcobehler.com/guides/a-guide-to-logging-in-java).

In short, if you just want to log statements to stdout, it's possible to add the following dependency: org.slf4j:slf4j-simple:1.7.29.
```
If you think something else is needed let me know.
